packages/nova-react/src/eventing/nova-eventing-provider.tsx

```diff
@@ -195,14 +195,11 @@ export const NovaEventingInterceptor: React.FunctionComponent<
   );
 
   // Internal should point to eventing/unmountEventing created by the interceptor, so that we can nest arbitrary numbers of interceptors
-  const internal: InternalEventingContext = React.useMemo(
```

Comment:
I assume the memo was there for a reason, no? But no tests that were covering its necessity, or are there tests for unwanted re-renders and we're still good?